lib/styles/_list.scss

```diff
@@ -18,6 +18,7 @@
     .link {
       color: #6bb3b7;
       text-decoration: none;
+      cursor: pointer;
```

Comment:
This needs to go in base-styles.scss otherwise it'll be overridden when we load in the custom styles.